DQMOffline/Trigger/plugins/METplusTrackMonitor.cc

```diff
@@ -0,0 +1,461 @@
+   setMETitle(metME_variableBinning_, "CaloMET [GeV]", "events / [GeV]");
+
+   histname = "metVsLS"; histtitle = "CaloMET vs LS";
+   bookME(ibooker, metVsLS_, histname, histtitle, ls_binning_.nbins, ls_binning_.xmin, ls_binning_.xmax, met_binning_.xmin, met_binning_.xmax);
+   setMETitle(metVsLS_, "LS", "CaloMET [GeV]");
+
+   histname = "metPhi"; histtitle = "CaloMET phi";
+   bookME(ibooker, metPhiME_, histname, histtitle, phi_binning_.nbins, phi_binning_.xmin, phi_binning_.xmax);
+   setMETitle(metPhiME_, "CaloMET #phi","events / 0.1 rad");
+
+   histname = "deltaphi_metjet1"; histtitle = "dPhi(CaloMET, jet1)";
+   bookME(ibooker, deltaphimetj1ME_, histname, histtitle, phi_binning_.nbins, phi_binning_.xmin, phi_binning_.xmax);
+   setMETitle(deltaphimetj1ME_, "#Delta#phi (CaloMET, j1)", "events / 0.1 rad");
+
+   histname = "metVsHltMet"; histtitle = "CaloMET vs hltMet";
+   bool bookDenominator = false;
+   bookME(ibooker, metVsHltMet_, histname, histtitle, met_binning_.nbins, met_binning_.xmin, met_binning_.xmax, met_binning_.nbins, met_binning_.xmin, met_binning_.xmax, bookDenominator);
+   setMETitle(metVsHltMet_, "hltMet (online) [GeV]", "CaloMET (offline) [GeV]", bookDenominator);
+
+   histname = "metVsHltMetClean"; histtitle = "CaloMET vs hltMetClean";
+   bookDenominator = false;
+   bookME(ibooker, metVsHltMetClean_, histname, histtitle, met_binning_.nbins, met_binning_.xmin, met_binning_.xmax, met_binning_.nbins, met_binning_.xmin, met_binning_.xmax, bookDenominator);
+   setMETitle(metVsHltMetClean_, "hltMetClean (online) [GeV]", "CaloMET (offline) [GeV]", bookDenominator);
+
+   // Track leg histograms
+
+   histname = "muonPt"; histtitle = "Muon PT";
+   bookME(ibooker, muonPtME_, histname, histtitle, pt_binning_.nbins, pt_binning_.xmin, pt_binning_.xmax);
+   setMETitle(muonPtME_, "Muon p_{T} [GeV]", "events / [GeV]");
+
+   histname = "muonPt_variable"; histtitle = "Muon PT";
+   bookME(ibooker, muonPtME_variableBinning_, histname, histtitle, muonPt_variable_binning_);
+   setMETitle(muonPtME_variableBinning_, "Muon p_{T} [GeV]", "events / [GeV]");
+
+   histname = "muonEta"; histtitle = "Muon eta";
+   bookME(ibooker, muonEtaME_, histname, histtitle, eta_binning_.nbins, eta_binning_.xmin, eta_binning_.xmax);
+   setMETitle(muonEtaME_, "Muon #eta", "events / 0.2");
+
+   histname = "deltaphi_muonmet"; histtitle = "dPhi(Muon, CaloMET)";
+   bookME(ibooker, deltaphimetmuonME_, histname, histtitle, phi_binning_.nbins, phi_binning_.xmin, phi_binning_.xmax);
+   setMETitle(deltaphimetmuonME_, "#Delta#phi (Muon, CaloMET)", "events / 0.1 rad");
+
+   histname = "mT_muonMet"; histtitle = "Transverse mass of muon-CaloMET system";
+   bookME(ibooker, mTmuonMetME_, histname, histtitle, pt_binning_.nbins, pt_binning_.xmin, pt_binning_.xmax);
+   setMETitle(mTmuonMetME_, "mT(Muon, CaloMET) [GeV]", "events / [GeV]");
+
+   histname = "muonPtVsLS"; histtitle = "Muon PT vs LS";
```

Comment:
you have already the MET vs LS, are you applying a different selection for this ?
if not, please drop